packages/tui/internal/commands/command.go

```diff
@@ -388,7 +387,8 @@ func LoadFromConfig(config *opencode.Config, customCommands []opencode.Command)
 		{
 			Name:        AppExitCommand,
 			Description: "exit the app",
-			Keybindings: parseBindings("ctrl+c", "<leader>q"),
+			// NOTE: ctrl+c requires a double press to exit while ctrl+d requires a single press
+			Keybindings: parseBindings("ctrl+c", "ctrl+d", "<leader>q"),
```

Comment:
This appears to be pulled directly into the user-facing UI for available commands, another reason I decided to continue using the existing `AppExitCommand`:

<img width="567" height="491" alt="Screenshot 2025-10-31 at 11 24 52 AM" src="https://github.com/user-attachments/assets/6b1367c2-0cdc-4c85-a3ac-34eb38bdc5eb" />